e2e/nx-plugin-e2e/tests/configuration.e2e.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'vitest';
+import { removeColorCodes } from '@code-pushup/test-utils';
+import { executeProcess } from '@code-pushup/utils';
+
+describe('nx-plugin g configuration', () => {
+  it('should run configuration generator on react-todos-app', async () => {
+    const { code, stderr, stdout } = await executeProcess({
+      command: 'nx',
+      args: [
+        'g',
+        '@code-pushup/nx-plugin:configuration react-todos-app --dryRun',
+      ],
+    });
```

Comment:
Shouldn't this be run via `npx`, in case `nx` isn't installed globally? (Same goes for `init.e2e.test.ts`)

```suggestion
    const { code, stderr, stdout } = await executeProcess({
      command: 'npx',
      args: [
        'nx',
        'g',
        '@code-pushup/nx-plugin:configuration react-todos-app --dryRun',
      ],
    });
```

Reply:
@BioPhoton Is there are some way I can prevent you from committing suggestion and ignoring comments like "Same goes for `init.e2e.test.ts`"? :sweat_smile: Maybe some emoji? I can also comment on every occurrence, but that would pollute the PR.

Reply:
I'm terribly sry!! I have it on my to improve list!! next to all the thing to consider while testing.
Please bare with me, I definitely want to respect your time.

What I have to do is NOT push the suggestion button before also open the IDE!

Reply:
In now